src/auth.rs

```diff
@@ -42,6 +42,10 @@
 //! An error will be returned if:
 //! - The calculated [`Tag`] does not match the expected.
 //!
```

Comment:
We should mention that an error will be returned if the secret key length is less than 32 or greater than 64.

Reply:
Fixed in https://github.com/brycx/orion/pull/88/commits/16d0e94a497661e16a74ce36d7edb48a358505df